ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/OptimisticHeadValidator.java

```diff
@@ -108,14 +108,14 @@ private void verifyOptimisticHeads() {
             });
   }
 
-  private SafeFuture<Void> reexecutePayload(
+  private SafeFuture<Void> recallNewPayload(
```

Comment:
I think `OptimisticHeadValidator` will wind up just getting deleted.  We don't expect the EL to execute blocks from non-canonical forks anymore, they just wind up being fully validated as part of the `forkChoiceUpdated` call if they ever become canonical.